### PR TITLE
chore(frontend): Spelling

### PR DESCRIPTION
**Pull request by jsoref**
### SUMMARY

Subset of #19648 per https://github.com/apache/superset/pull/19648#pullrequestreview-939431828

### ADDITIONAL INFORMATION

- [ ] Has associated issue:
- [ ] Required feature flags:
- [ ] Changes UI
- [ ] Includes DB Migration (follow approval process in [SIP-59](https://github.com/apache/superset/issues/13351))
  - [ ] Migration is atomic, supports rollback & is backwards-compatible
  - [ ] Confirm DB migration upgrade and downgrade tested
  - [ ] Runtime estimates and downtime expectations provided
- [ ] Introduces new feature or API
- [ ] Removes existing feature or API

### CODE REVIEW

File: docs/static/resources/openapi.json
Comment (ktmud) [inline docs/static/resources/openapi.json:15621]: Isn't this file programmatically generated?

Comment (jsoref) [reply docs/static/resources/openapi.json:15621]: It appears to be updated by commits that change things: https://github.com/apache/superset/commit/02ef9ca4cd60d0d05f0afd2cebc1cabb83dfd4df

The relevant items should be changed below...

Comment (nytai) [reply docs/static/resources/openapi.json:15621]: unfortunately I think this is part of the api, you would have to update flask appbuilder to fix it and it would be a breaking change. @dpgaspar

Comment (jsoref) [reply docs/static/resources/openapi.json:15621]: I'm quite happy to drop anything that proves to be problematic. It's not a big deal.

And in the interest of moving things forward while I'm away, I'll drop this now.

You're welcome to fix it w/o me, or I can look at it once the rest merges.

Comment (ktmud) [reply docs/static/resources/openapi.json]: https://github.com/apache/superset/blob/3313530f4de2d1cf1719d6b2beda89a5bfbba2cd/superset/cli/update.py#L148-L152

This file is generated by a script. There is a superset command: `superset update-api-docs` but unfortunately it is not documented anywhere.

**Comment by rusackas** [comment]
@jsoref much like your other PR, we really appreciate cleaning up all these typos! Would you be able to remove the auto-generated files and rebase to mitigate the conflicts? Hopefully we can usher this through (CC @amitmiran137 for possible assistance here).

**Comment by jsoref** [comment]
This is a strict rebase. Lemme see about finding the autogenerated files and dropping them (notably, i think that some linters are complaining -- I suspect that I'd rather someone else make the linters happy, I don't have infinite time)

**Comment by jsoref** [comment]
The item that was listed was already dropped. I think I left the item unresolved because in theory someone should eventually address it somewhere.

**Review by rusackas — APPROVED** [review]
LGTM! Thanks for all of this! We've gotta add this tooling to our CI flow...

File: superset-frontend/src/views/CRUD/welcome/Welcome.tsx
Comment (rusackas) [inline superset-frontend/src/views/CRUD/welcome/Welcome.tsx:231]:
```suggestion
    getRecentActivityObjs(user.userId!, recent, addDangerToast, otherTabFilters)
```

**Comment by rusackas** [comment]
Looks like this'll need a codeowner approval from @michael-s-molina @geido @ktmud as well.

**Review by michael-s-molina — APPROVED** [review]

**Pull request merged**
